browser-extension/src/addTournesolRecommendations.js

```diff
@@ -257,3 +265,40 @@ function loadRecommandations() {
     handleResponse
   );
 }
+
+function millifyViews(videoViews) {
+  // Intl.NumberFormat object is in-built and enables language-sensitive number formatting
+  return Intl.NumberFormat('en', { notation: 'compact' }).format(videoViews);
+}
+
+function viewPublishedDate(publishedDate) {
+  const date1 = new Date(publishedDate);
+  const date2 = new Date();
+  // we will find the difference in time of today's date and the published date, and will convert it into Days
+  // after calculating no. of days, we classify it into days, weeks, months, years, etc.
+  const diffTime = date2.getTime() - date1.getTime();
+  const diffDays = Math.floor(Math.abs(diffTime / (1000 * 3600 * 24)));
```

Comment:
If the published date is 3 days *after* the user's system date I think we don't want to return "3 days ago" like the code currently does.

Maybe we should just return an empty string.

Reply:
I can add a clause for displaying an empty string if the `diffDays` returns a negative value.

Reply:
The fix doesn't work because it still uses the absolute value of `diffTime`.

Reply:
You are right, I overlooked the logic, will make the changes accordingly.